spectests/structs_test.go

```diff
@@ -22,37 +22,46 @@ type codec interface {
 	ssz.Marshaler
 	ssz.Unmarshaler
 	ssz.HashRoot
+
+	// TODO
+	GetTreeWithWrapper(w *ssz.Wrapper) (err error)
+	GetTree() (*ssz.Node, error)
 }
 
 type testCallback func() codec
 
 var codecs = map[string]testCallback{
-	"AggregateAndProof":       func() codec { return new(AggregateAndProof) },
```

Comment:
I did comment this cases to test the GetTree separately, could you uncomment it?